src/screens/SUE/SueListScreen.tsx

```diff
@@ -105,17 +98,17 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const { showViewSwitcherButton = false } = sueListItem;
   const query = route.params?.query ?? '';
 
-  const limit = 20;
-  const initial_start_date = '1900-01-01T00:00:00+01:00';
-  const dataCountQueryVariables = {
-    start_date: initial_start_date
-  };
   const initialQueryVariables = route.params?.queryVariables ?? {
-    initial_start_date,
     limit,
-    offset: 0
+    offset: 0,
+    ...initial_start_date
   };
   const [queryVariables, setQueryVariables] = useState(initialQueryVariables);
+  const [dataCountQueryVariables, setDataCountQueryVariables] = useState(() => {
+    const { limit, offset, ...rest } = queryVariables;
+
+    return { ...rest, ...initial_start_date };
```

Comment:
if initial start date is already part of the initial query variables (line 104), which is the initial value for the query variables (line 106), than the start date is already in `rest`, or am i wrong?

Reply:
what about 

```
useState({
  ...queryVariables,
  limit: undefined,
  offset: undefined
})
```

Reply:
We use `initialQueryVariables` in two parts. We need `initialQueryVariables` to reset the filter. This way, we can reset the filter.

Reply:
did you comment right? this conversation is not about resetting

Reply:
[171b753](https://github.com/smart-village-solutions/smart-village-app-app/pull/1252/commits/171b75321c0c919670bb73f35fc32a6465171a21)